From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?ZX=E5=A4=8F=E5=A4=9C=E4=B9=8B=E9=A3=8E?=
 <snwcreations@qq.com>
Date: Sat, 21 Sep 2024 19:42:27 +0800
Subject: [PATCH] fix: add dummy implementation of NMSImpl delegating to
 upstream version

This commit should make the NMS bridge initialization successful

diff --git a/v1_20_R4/src/main/java/net/citizensnpcs/nms/v1_20_R3/util/NMSImpl.java b/v1_20_R4/src/main/java/net/citizensnpcs/nms/v1_20_R3/util/NMSImpl.java
new file mode 100644
index 0000000000000000000000000000000000000000..f425f985bd2a22416a88cb74564f3211e1a79391
--- /dev/null
+++ b/v1_20_R4/src/main/java/net/citizensnpcs/nms/v1_20_R3/util/NMSImpl.java
@@ -0,0 +1,5 @@
+package net.citizensnpcs.nms.v1_20_R3.util;
+
+// Dummy class used to make NMS bridge initialization successful
+public class NMSImpl extends net.citizensnpcs.nms.v1_20_R4.util.NMSImpl {
+}
